compiler/rustc_session/src/options.rs

```diff
@@ -810,6 +810,8 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
     target_feature: String = (String::new(), parse_target_feature, [TRACKED],
         "target specific attributes. (`rustc --print target-features` for details). \
         This feature is unsafe."),
+    tune_cpu: Option<String> = (None, parse_opt_string, [TRACKED],
+        "select processor to schedule for (`rustc --print target-cpus` for details)"),
```

Comment:
This should start out as a debugging option, otherwise it would be instantly stable

Reply:
Makes sense. Let me see how that works...